cocos/rendering/custom/define.ts

```diff
@@ -545,6 +545,7 @@ export function buildForwardPass (camera: Camera,
     }
     if (!isOffScreen) {
         ppl.updateRenderWindow(forwardPassRTName, camera.window);
+        ppl.updateDepthStencil(forwardPassDSName, width, height);
```

Comment:
`forwardPassDS` renderpass desc not updated, so managed depthStencil texture not re-created.
gfx::vulkan creates frame buffer with `foreach(attachment) fb.extent = min(fb.extent, attachment.extent)`.